bundles/org.openhab.voice.rustpotterks/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -7,6 +7,7 @@
 	<name>Rustpotter Keyword Spotter</name>
 	<description>This voice service allows you to use the open source library Rustpotter as your keyword spotter in
 		openHAB.</description>
+	<connection>local</connection>
```

Comment:
Same than for Vosk, I think that connection can be none, Rustpotter library is included in the bundle and executed locally by the add-on.